### ISSUE TITLE
Update documentation for manually assessing a webpage for WCAG conformance

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#5954]
We have outdated documentation for conducting an accessibility audit on a web page:

https://wellcomecloud.sharepoint.com/:f:/r/sites/wcdigitalexperience/Shared%20Documents/Accessibility?csf=1&web=1&e=ufLyLm

We need to update the guidance to work with current browser add ons and to include new guidelines from WCAG 2.1

**Comment by jennpb** [comment]
@gestchild Is this still needed?

**Comment by gestchild** [comment]
@jennpb probably not, it was guidance on doing the type of thing Hassell just did for us and I think it's better that specialists do it for us as long as we can make it a regular thing.

It may be useful if we want to test more things ourselves but probably not high priority given the above.

**Comment by jennpb** [comment]
I agree it would be useful to do regular 3rd party assessments. I'll close for now and include that recommendation in my handover notes.
